samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -235,6 +239,91 @@ static void subgroup_send(struct stream_index stream_idx)
 	prev_ret = ret;
 }
 
+static int dummy_data_config(const struct shell *shell, uint8_t big_index, uint8_t sub_index,
+			     uint8_t bis_index)
+{
+	int ret;
+
+	if (sd_card_not_present) {
+		LOG_DBG("No SD card, using dummy data");
+	} else {
+		shell_error(shell, "Failed as SD card present");
```

Comment:
Mix of "normal" logging and shell logging. I think we should stick with just normal logging here as this is an automatic function and not something that was prompted by the user.